.github/workflows/cypress.yml

```diff
@@ -0,0 +1,111 @@
+name: Cypress
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - stable*
+
+env:
+  APP_NAME: deck
+  CYPRESS_baseUrl: http://nextcloud.local/index.php/
+
+jobs:
+  cypress:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: [14.x]
+        # containers: [1, 2, 3]
+        php-versions: [ '7.4' ]
+        databases: [ 'sqlite' ]
+        server-versions: [ 'master' ]
```

Comment:
**Question:** 
We are trigging this test in `stable*` branches too.
But the server version always is master, there is a way to test stable branches with the stable version of the server?